proposals/type-aliases.md

```diff
@@ -221,6 +289,7 @@ fun throws(): Empty = ...       // Error: return type Nothing should be specifie
 ```
 
 ### Type aliases as values
```

Comment:
Why? This is the terminology we use in resolve.
Probably "Type alias companion object" would be more clear.